requirements.txt

```diff
@@ -1,3 +1,3 @@
-colorama>=0.3.7
-pandas>=0.19.2
-pyyaml>=3.12
+colorama==0.3.7
+pandas
+pyyaml==3.11
```

Comment:
I'm also fine with just ditching the requirements files entirely. The only ones really being used now are the dev & test ones, just for the Travis build. Those could just as easily be enumerated directly in the Travis config.